include/spirv/spir-v.xml

```diff
@@ -64,15 +64,15 @@
         <id value="11"  vendor="Intel"      comment="Contact Alexey, alexey.bader@intel.com"/>
         <id value="12"  vendor="Imagination" comment="Contact Stephen Clarke, stephen.clarke@imgtec.com"/>
         <id value="13"  vendor="Google" tool="Shaderc over Glslang" comment="Contact David Neto, dneto@google.com"/>
-        <id value="14"  vendor="Google" tool="spiregg" comment="Contact Lei Zhang, antiagainst@google.com"/>
+        <id value="14"  vendor="Google" tool="spiregg" comment="Contact Lei Zhang, antiagainst@gmail.com"/>
```

Comment:
Google still works on this, and Lei doesn't. 
I'm confirming with internal folks about who is best to take over being point of contact.

Reply:
You can add me: stevenperron@google.com